src/plugins/remark-button.ts

```
@@ -0,0 +1,88 @@
+  };
+  children: PhrasingContent[];
+}
+
+// Union type for the directive nodes this plugin handles
+type DirectiveNode = ContainerDirective | LeafDirective | TextDirective;
+
+function h(
+  el: string,
+  attrs: HastProperties = {},
+  children: PhrasingContent[] = []
+): MdastLinkNode {
```

ESLint

[error] 'MdastLinkNode' is not defined.

(no-undef)